Makefile

```diff
@@ -0,0 +1,14 @@
+default: image
+
+all: image
+
+image:
+	docker build . \
+	-f Dockerfile \
+	--tag matplotlib/mpl-docker:debug-latest
+
+run:
+	docker run --rm -it \
+	-v $(shell pwd):/mpl_source \
```

Comment:
But you should be mounting the Matplotlib source, not this directory.

Reply:
No, not for testing the build of this image.

Reply:
There's nothing in this repo to test with.

Reply:
The `Makefile` is local to this project. What I mean is that if you're working on refactoring the image and you want to build things locally and then drop into it to look around and play then this `Make` target is helpful. For use in actually doing things in `matplotlib` that's a different story. But what @tacaswell [is demoing in PR #5](https://github.com/matplotlib/mpl-docker/pull/5#issuecomment-657091073) wouldn't ever be using this `Makefile`. That would need some shell alias or something.

Sorry, I may have a different idea in my head about what the intended use case of this specific repository is. I thought that this was for just developing the Docker image for a canonical matplotlib build/dev environment. Am I on the same train of thought as you, or did I get confused?

Reply:
oh sorry! I think that my choice of `mpl_source` is confusing here(?). I know that this doesn't contain the matplotlib source code, but just thought that as this would be used for matplotlib source that I'd hint at what this image would be used for. If this is bad I'll change it.